cubie_setup.md

```diff
@@ -0,0 +1,102 @@
+Throughout this document I'll refer to the wireless interface as `wlan1`; on a CubieBoard, it's likely that the wireless dongle will be `wlan0` instead.
+
+* install hostapd (for wireless access point) and dnsmasq (for dhcp server): 
+
+```
+sudo apt-get install hostapd dnsmasq
+```
+
+* apt will start dnsmasq right after it's installed.  You don't want it to be running yet as it's not properly configured, so kill it:
+
+```
+sudo kill `pidof dnsmasq`
+```
+
+* create the file `/etc/hostapd.conf` and add these lines to it:
+
+```
+interface=wlan1
+ssid=cubietruck-test
+channel=11
+```
+
+* initialise the wireless usb dongle:
+
+```
+sudo ip link set wlan1 up
+```
+
+* start hostapd:
+
+```
+sudo hostapd /etc/hostapd.conf
+```
+
+Output should look something like this (the important part being "AP-ENABLED"):
+```
+Configuration file: /etc/hostapd.conf
+Using interface wlan1 with hwaddr c4:04:15:7a:98:a4 and ssid "cubietruck-test"
+VLAN: vlan_set_name_type: SET_VLAN_NAME_TYPE_CMD name_type=2 failed: Package not installed
+wlan1: interface state UNINITIALIZED->ENABLED
+wlan1: AP-ENABLED 
```

Comment:
You weren't, but my instructions were!  Starting `hostapd` with `-B` will background the process and give you your terminal back; updated. Thank you!